lib/FolderRow/FolderRow.js

```diff
@@ -27,10 +29,15 @@ const FolderRow = ({
     'folder-row--highlight': highlight
   });
 
+  const folderNameClassNames = cx('folder-row__name', {
+    'h-width-100': isEditing
```

Comment:
👍 so the input spans correctly but not until someone is editing so hover isn't affected.